src/Samples/VisualBasic/ConsoleClassifier/Program.vb

```diff
@@ -21,10 +21,10 @@ Sub Main()
 WriteLine(""Hello, World!"")
 End Sub
 End Module")
-        document = Await Formatter.FormatAsync(document)
-        Dim text As SourceText = Await document.GetTextAsync()
+        document = Await Formatter.FormatAsync(document).ConfigureAwait(False)
```

Comment:
Why don't we run on samples? My intuition, based on the setup of our build, is that it would run an samples.